docs/user/metrics/features.csv

```diff
@@ -8,3 +8,4 @@ Library,ML Backend,I/O Backend,Spatial Backend,Transform Backend,Datasets,Weight
 `TerraTorch`_,PyTorch,"GDAL, h5py, pandas, xarray",R-tree,Albumentations,22,1,✅,✅,❌,🚧
 `SITS`_,R Torch,GDAL,-,tidyverse,22,0,❌,✅,✅,✅
 `scikit-eo`_,"scikit-learn, TensorFlow","pandas, scipy, numpy, rasterio","geopandas",numpy,0,0,❌,❌,❌,🚧
+`otbtf`_,TensorFlow,"GDAL, OTB",-,-,0,0,✅,✅,❌,❌
```

Comment:
* **Library**: Should we use OTBTF instead of otbtf? Not sure if it's more common to capitalize the name or not.
* **Spatial Backend**: Do you use any kind of software for spatiotemporal indexing of data? Maybe GDAL does I/O and OTB does spatiotemporal indexing?
* **Transform Backend**: Does OTBTF support data augmentation/transforms? If so, what library do you use for it? I see you have one-hot encodings done natively in TF, maybe that's the only transform? If so I would just list TensorFlow.
* **CLI**: Is there any documentation on your command-line interface? I didn't see anything in the docs or code.

Reply:
- Library: lowercase/uppercase are both used but we can use OTBTF if you prefer
- Spatial Backend: no indexing of data, OTBTF is a kind of "low level" library providing building blocks as OTB applications, it is up to the user to do everything
- Transform backend: OTBTF does not provide anything for data augmentation, OTB and GDAL handles all the spatial/projections transformations (resampling, etc). There are only a few layers in OTBTF to ease the pain of users (like one_hot) but not so much
- CLI: we used to copy/paste the `--help` output of CLI in the doc, but we rapidly have stopped to to that since it can be out of sync and confuse users

Reply:
Yeah, let's go with OTBTF consistently everywhere.